spec/controllers/articles_controller_spec.rb

```diff
@@ -37,76 +30,83 @@
 
-      let(:article) { create(:article) }
+        let(:article) { create(:article) }
 
-      let(:params) { {article: {title: article.title,
-                                content: article.content}} }
+        let(:params) { {article: {title: article.title,
+                                  content: article.content}} }
 
-      before(:each) do
-        allow(controller).to receive(:current_user).and_return(article.user)
-      end
+        before(:each) do
+          allow(controller).to receive(:current_user).and_return(article.user)
+        end
 
-      it 'saves a new article to the database' do
-        expect {
+        it 'saves a new article to the database' do
+          expect {
+            post :create, params: params
+          }.to change(Article, :count).by(1)
+        end
+
+        it 'displays a success message' do
           post :create, params: params
-        }.to change(Article, :count).by(1)
-      end
+          expect(response.request.flash[:success]).to_not be_nil
+        end
 
-      it 'displays a success message' do
-        post :create, params: params
-        expect( response.request.flash[:success] ).to_not be_nil
+        it 'redirects to homepage' do
+          post :create, params: params
+          expect(response).to redirect_to root_path
+        end
       end
 
-      it 'pending: displays the article post'
+      context 'without a title included in params' do
+        let(:article) { create(:article) }
 
-      it 'redirects to homepage' do
-        post :create, params: params
-        expect(response).to redirect_to root_path
-      end
-    end
+        before(:each) do
+          allow(controller).to receive(:current_user).and_return(article.user)
+        end
 
-    context 'without a title included in params' do
-      let(:article) { create(:article) }
+        let(:params) { {article: {title: nil,
+                                  content: article.content}} }
 
-      before(:each) do
-        allow(controller).to receive(:current_user).and_return(article.user)
-      end
-
-      let(:params) { {article: {title: nil,
-                                content: article.content}} }
-
-      it 'does not save a new article to the database' do
-        expect {
+        it 'does not save a new article to the database' do
+          expect {
+            post :create, params: params
+          }.to_not change(Article, :count)
+        end
+        it 're-renders #new' do
           post :create, params: params
-        }.to_not change(Article, :count)
-      end
-      it 're-renders #new' do
-        post :create, params: params
-        expect(response).to render_template "new"
+          expect(response).to render_template "new"
+        end
       end
-    end
 
-    context 'without content included in params' do
+      context 'without content included in params' do
 
-      let(:article) { create(:article) }
+        let(:article) { create(:article) }
```

Comment:
Same as comment above:
```suggestion
        let(:article) { create(:article, content: nil) }
```